spec/webpush_spec.rb

```diff
@@ -43,18 +43,28 @@
 
       result = subject
 
-      expect(result).to be_a(Net::HTTPCreated)
-      expect(result.code).to eql('201')
```

Comment:
Pertinent to my comment above, I'd prefer we keep these tests in place for the "happy path": returning the `response` object instead of `nil`.